www/src/components/layer-model/image-model/image-content-sections.js

```diff
@@ -0,0 +1,154 @@
+/** @jsx jsx */
+import { jsx } from "theme-ui"
+import { Link } from "gatsby"
+import {
+  LayerContentWrapper,
+  ExampleWrapper,
+  CodeWrapper,
+} from "../model-wrapper"
+
+const InstallLayerContent = ({ index, displayCodeFullWidth }) => (
+  <LayerContentWrapper
+    key={`content-wrapper${index}`}
+    index={index}
+    displayCodeFullWidth={displayCodeFullWidth}
+  >
+    <ExampleWrapper>
+      <CodeWrapper title="shell" language="shell">
+        {`npm install gatsby-transformer-sharp gatsby-plugin-sharp gatsby-image`}
+      </CodeWrapper>
+    </ExampleWrapper>
+    <div>
+      <p>Using images in Gatsby requires a few different plugins.</p>
+      <p>
+        <b>Install</b> <code>gatsby-transformer-sharp</code> and{" "}
+        <code>gatsby-plugin-sharp</code> to transform and process your images.{" "}
+        <code>gatsby-image</code> is a package that includes a component you'll
+        use later in "View".
+      </p>
+    </div>
+  </LayerContentWrapper>
+)
+
+const ConfigLayerContent = ({ index, displayCodeFullWidth }) => (
+  <LayerContentWrapper
+    key={`content-wrapper${index}`}
+    index={index}
+    displayCodeFullWidth={displayCodeFullWidth}
+  >
+    <ExampleWrapper>
+      <CodeWrapper title="gatsby-config.js" language="javascript">
+        {`plugins: [
+  \`gatsby-transformer-sharp\`,
+  \`gatsby-plugin-sharp\`,
+  {
+    resolve: \`gatsby-source-filesystem\`,
+    options: {
+      path: \`\${__dirname}/src/images\`,
+    },
+  },
+  ...
+]`}
+      </CodeWrapper>
+    </ExampleWrapper>
+    <div>
+      <p>
+        When you set up the <b>Config</b> for your site, you'll want to include{" "}
+        <code>gatsby-transformer-sharp</code> and{" "}
+        <code>gatsby-plugin-sharp</code>.
+      </p>
+      <p>
+        You'll also need to configure <code>gatsby-source-filesystem</code> with
+        the appropriate options. The <code>path</code> should be the path to the
+        directory your image files live in.
+      </p>
+    </div>
+  </LayerContentWrapper>
+)
+
+const QueryLayerContent = ({ index, displayCodeFullWidth }) => (
+  <LayerContentWrapper
+    key={`content-wrapper${index}`}
+    index={index}
+    displayCodeFullWidth={displayCodeFullWidth}
+  >
+    <ExampleWrapper>
+      <CodeWrapper title="src/pages/index.js" language="graphql">
+        {`const query = graphql\`
+  query {
+    file(relativePath: { eq: "images/gatsby-logo.png" }) {
+      childImageSharp {
+        fluid {
+          ...GatsbyImageSharpFluid
+        }
+      }
+    }
+  }
+\`
+`}
+      </CodeWrapper>
+    </ExampleWrapper>
+    <div>
+      <p>
+        In order to take advantage of fast image processing in Gatsby, you need
```

Comment:
The word fast here seems a little ambiguous to me, is it the processing/transforming of the images that's fast, the load times?

Maybe saying something like:

```suggestion
        In order to take advantage of the optimized image sizes generated by Gatsby, you need
```

🤷‍♂